frontend/src/static/js/v_zoom.js

```diff
@@ -130,6 +130,10 @@ window.vZoom = {
       this.selectedFileTypes = this.fileTypes.slice();
     },
 
+    getFontColor(color) {
+      return window.getFontColor(color);
+    },
+
```

Comment:
is `this.getFontColor` really big trouble? seems now you have to duplicate the common function for all subcomponent.

Reply:
A problem for projectify frontend, since `this.` does use the variables from `window.`

If the duplication is big trouble I can try using mixins.

Reply:
Yes, could you please try to use mixins for this? If we really gonna projectify this project properly, mixin will eventually be required.

Reply:
Actually after trying mixins, I don't think they are a good idea. Manually importing the required functions in would probably be a better choice for easier maintenance. I did change the implementation a little to make it less wordy though.